pkg/cmd/create/create.go

```diff
@@ -0,0 +1,71 @@
+// Copyright © 2020 The Tekton Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package create
+
+import (
+	"log"
+	"os"
+	"os/exec"
+
+	"github.com/spf13/cobra"
+)
+
+// runCmd is suitable for use with cobra.Command's Run field
+type runCmd func(*cobra.Command, []string)
+
+func Command() *cobra.Command {
+	eg := `Create resources defined in foo.yaml in namespace 'bar':
+
+	tkn create -f foo.yaml -n bar
+`
+
+	c := &cobra.Command{
+		Use:          "create",
+		Short:        "Uses kubectl create to create Kubernetes resources",
+		Example:      eg,
+		SilenceUsage: true,
+		Annotations: map[string]string{
+			"commandType": "main",
+		},
+		Run: invokeKubectl("kubectl"),
+		FParseErrWhitelist: cobra.FParseErrWhitelist{
+			UnknownFlags: true,
+		},
+	}
+
+	return c
+}
+
+// returns a runCmd that simply passes CLI arguments
+// through to a binary named command.
+func invokeKubectl(command string) runCmd {
```

Comment:
What do you think about adding a check to look out for the binary and let the user that she would need to install `kubectl` before executing this command?

Reply:
Agreed. Let me add a check for `kubectl` binary.

Reply:
Check added via `isKubectlAvailable()`